src/solver.cpp

```diff
@@ -303,6 +303,12 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     RegisterWithSolver(
         registry, ++id, ConvHipImplicitGemmBwdDataV1R1Xdlops{}, miopenConvolutionAlgoImplicitGEMM);
 
+    RegisterWithSolver(
```

Comment:
Please look at comment at line 205 and fix this

Reply:
Hi @atamazov I changed the order, please help review again